ui/src/styles/_global.scss

```diff
@@ -28,6 +28,9 @@ body {
 }
 
 main {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
```

Comment:
this would make it so if for example a user was on mobile and there isnt much content, that the user would have to scroll just to see the footer since min height is 100%. To simulate pushing the footer to the bottom but not necessarily maxing a min height, try looking at something like this https://dev.to/domysee/keeping-the-footer-at-the-bottom-with-css-flexbox-5h5f